api/src/models/FedoraObject.ts

```diff
@@ -123,6 +123,14 @@ export class FedoraObject {
         );
     }
 
+    async addParentRelationship(parentPid: string): Promise<void> {
+        return this.addRelationship(
+            "info:fedora/" + this.pid,
```

Comment:
"info:fedora/" is the same for all parameters. You can maybe add this to part of the base url.

Reply:
In this instance, we're setting up an RDF triple where all three parts happen to use info:fedora/ URIs. However, this is not universally guaranteed. It wouldn't make sense to hard-code info:fedora into the generic addRelationship method, and I don't see an obvious way to simplify what we have here. I think the redundancy may be unavoidable.